### PR TITLE
Created the create account panel

### PR DESCRIPTION
**Pull request by Jramos8**
Added the functionality for the cancel button to return to the main login panel.

### CODE REVIEW

**Comment by sambutler1017** [comment]
Need to pull in master

File: Airline_Project_Group_26/BaseForm.Designer.cs [hunk 1]
```diff
@@ -41,23 +41,44 @@ private void InitializeComponent()
...
             this.label1 = new System.Windows.Forms.Label();
             this.home_panel = new System.Windows.Forms.Panel();
-            this.home_page_content = new System.Windows.Forms.Panel();
-            this.label2 = new System.Windows.Forms.Label();
-            this.profile_panel_content = new System.Windows.Forms.Panel();
-            this.label6 = new System.Windows.Forms.Label();
             this.side_navbar = new System.Windows.Forms.TableLayoutPanel();
             this.navbar_panel = new System.Windows.Forms.Panel();
             this.label5 = new System.Windows.Forms.Label();
-            this.profile_icon_navbar = new System.Windows.Forms.PictureBox();
```
Comment (sambutler1017) [inline Airline_Project_Group_26/BaseForm.Designer.cs:51]: Here. There is where you deleted the profile icon

**Review by sambutler1017 — CHANGES_REQUESTED** [review]

File: Airline_Project_Group_26/BaseForm.Designer.cs [hunk 2]
```diff
@@ -206,148 +237,374 @@ private void InitializeComponent()
...
+            this.label5.Margin = new System.Windows.Forms.Padding(4, 0, 4, 0);
             this.label5.Name = "label5";
-            this.label5.Size = new System.Drawing.Size(194, 26);
+            this.label5.Size = new System.Drawing.Size(235, 31);
             this.label5.TabIndex = 4;
             this.label5.Text = "Airline Scheduler";
             // 
-            // profile_icon_navbar
-            // 
-            this.profile_icon_navbar.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Right)));
```
Comment (sambutler1017) [inline Airline_Project_Group_26/BaseForm.Designer.cs:314]: This also needs added back in

**Review by sambutler1017 — CHANGES_REQUESTED** [review]

File: Airline_Project_Group_26/BaseForm.cs [hunk 3]
```diff
@@ -58,5 +60,14 @@ public partial class BaseForm : Form  {
...
         /// <param name="e">the event arguments</param>
         private void profile_icon_navbar_Click(object sender, EventArgs e) => homePanel.onProfileClick();
+
+        /**
+         * When the user wants to cancel the creation of an account this method will get called 
+         * 
+         * @param sender - where the object was clicked from
+         * @param e - the event arguments
+         */
+        private void cancel_button_Click(object sender, EventArgs e) => createAccountPanel.onCancelClick();
```
Comment (sambutler1017) [inline Airline_Project_Group_26/BaseForm.cs:70]: These are java docs. Look at the method above for document commenting.

**Review by sambutler1017 — CHANGES_REQUESTED** [review]

**Review by ischank1 — APPROVED** [review]

**Review by cheng801 — APPROVED** [review]

**Pull request merged**